maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -185,7 +188,13 @@ private void loadProjectProperties(Builder builder, MavenProject project) throws
 
 		// Merge in current project properties
 		File baseDir = project.getBasedir();
-		File bndFile = new File(baseDir, Project.BNDFILE);
+		File bndFile;
+		if(bndLocation != null) {
+			bndFile = new File(bndLocation);
```

Comment:
Ok, I'm clearly missing a lot of details about the plugin. Any hints on how this could be implemented correctly?